Makefile.toml

```diff
@@ -119,6 +119,14 @@ AMI_DATA_FILE_SUFFIX = "amis.json"
 # The type of testsys test that should be run.
 # `quick` will run a quick test which usually tests that the instances are reachable.
 # `conformance` will run a certified conformance test, these tests may take up to 3 hrs.
+# `migration` will run an upgrade downgrade test including:
+#    1: an initial quick test
```

Comment:
nit. I would refer to quick test as `quick` test, just so its obvious that `quick` is the type of the quick test you are running.